pywt/tests/test_multidim.py

```diff
@@ -88,8 +88,9 @@ def test_byte_offset():
 
         for mode in pywt.Modes.modes:
             expected = pywt.dwtn(data, wavelet)
-            padded = np.ones((3, 6), dtype=np.dtype([('data', data.dtype),
-                                                     ('pad', 'byte')]))
+            padded = np.ones((3, 6), dtype=np.dtype({'ddata': (data.dtype, 0),
```

Comment:
Can you change this field name from 'ddata' back to 'data'?  It is causing one test failure as is